daemon/cmd/daemon_main.go

```diff
@@ -6,6 +6,7 @@ package cmd
 import (
 	"context"
 	"fmt"
+	"github.com/cilium/cilium/pkg/maps/egressmap"
```

Comment:
Can you move this down below together with all the other cilium imports please?

Reply:
Thanks and done.